### DISCUSSION TITLE
Are metrics suitable for 2D data or 3D data?

### DISCUSSION THREAD

**Discussion by lxy51** [francois-rozet/piqa#37] [Q&A] [answered]
Which metrics are suitable for 2D data or 3D data?

**Comment by francois-rozet** [comment, marked as answer]
Hello @lxy51, thank you for your question. Most of the metrics are designed for images, that is tensors with shape $C \times H \times W$. They cannot be applied to videos, that is tensors with shape $C \times L \times H \times W$.

There are two exceptions: `SSIM` and `PSNR`. These will work with videos/3D data. It is also possible to apply the other metrics to individual frames in videos.

**Reply by lxy51** [reply]
Okay, thank you.
